### PR TITLE
Added usage instructions

### PR DESCRIPTION
**Pull request by ExoticObjects**
Added detailed usage instructions to potentially help others avoid some of the pitfalls that got me...

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+# Possible Gotchas
+**No headers created**
+This can happen if your spec does not lint. The packager may successfully complete and create your framework, but there will be no headers. Without headers, nothing will work, so...
+
+**The `source_files` pattern did not match any file**
+The packager creates a podspec with these lines:
+
+```ruby
+s.platform = :ios, '9.0' 
+s.ios.platform = :ios, '9.0'
```
Comment (segiddins) [inline USAGE.md]: this line is a misuse of the podspec DSL, and the packager generating it is _definitely_ a bug

Comment (ExoticObjects) [inline USAGE.md]: which line - s.platform or s.ios.platform?

Comment (segiddins) [inline USAGE.md]: `s.ios.platform` -- don't worry, I'm about to PR a fix

**Comment by ExoticObjects** [comment]
Thoughts on this?

**Comment by neonichu** [comment]
@ExoticObjects Thanks for this, but in the current form feels more like a blog post than actual usage instructions. Maybe you want to post it to your blog and the README could link to it as a "Further reading" kind of thing.

**Comment by ExoticObjects** [comment]
Fair enough, will do.

**Comment by neonichu** [comment]
Let's close this.

**Pull request closed without merging**